pages/content/posts/improve-formal-syntax/index.md

```diff
@@ -0,0 +1,37 @@
+---
+author: "Will Bamberg"
+title: "Improving the CSS formal syntax"
+date: "2022-10-06"
+tags: ["projects"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "How we made the CSS formal syntax more readable and reliable, with the help of some great open source software packages."
+---
+
+If you've spent much time with our CSS docs, you've probably seen the "Formal syntax" section in the CSS property reference pages. When we ask developers what they think of this part of the docs, most of them say something like "I have no idea what this means". This used to be my reaction too, but now I think it can be extremely useful. CSS properties can have very complex syntax, and explaining it in prose can be really hard. Once you've learned a few rules, the formal syntax gives you a very precise description of the syntax in a very concise manner.
```

Comment:
```suggestion
If you've spent much time with MDN's CSS docs, you've probably seen the "Formal syntax" section in the CSS property reference pages (see for example the [`background` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/background#formal_syntax)). When we ask developers what they think of this part of the docs, most of them say something like "I have no idea what this means". This used to be my reaction too, but now I think it can be extremely useful. CSS properties can have very complex syntax, and explaining it in prose can be really hard. Once you've learned a few rules, the formal syntax gives you a very precise description of the syntax in a very concise manner.
```

Reply:
```suggestion
If you've spent much time with the CSS docs on MDN, you've probably seen the ["Formal syntax" section](https://developer.mozilla.org/en-US/docs/Web/CSS/background#formal_syntax) in the CSS property reference pages. When we ask developers what they think of this part of the docs, most of them say something like "I have no idea what this means". This used to be my reaction too, but now I think it can be extremely useful. CSS properties can have very complex syntax, and explaining it in prose can be really hard. Once you've learned a few rules, the formal syntax gives you a very precise description of the syntax in a very concise manner.
```

is that OK? Avoiding the possessive "MDN's" which is a bit ugly, and making the link part more flowy.